ranger/config/commands.py

```diff
@@ -1959,3 +1935,40 @@ def tab(self, tabnum):
             in sorted(self.modes.keys())
             if mode
         )
+
+
+class yank_content(Command):
+    """:yank_content
+
+    Copies the file's contents into both the primary X selection and
+    the clipboard. The command does not work on bianry files,
+    directoroies and mutliple selection.
+
+    """
+
+    def execute(self):
+        import subprocess
+        from ranger.ext.clipboard_util import clipboards
+
+        select = self.fm.thistab.get_selection()
+
+        if len(select) != 1:
+            self.fm.notify('Cannot yank content from selection of files')
```

Comment:
Could be cool to copy multiple files at the same time maybe. For example to pastebin a problem description, select your relevant files, `yc`, open up an editor, paste, edit a bit maybe adding headers and a description.